Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -137,21 +138,33 @@ public struct WorldScaleGeoTrackingSceneView: View {
         .task {
             do {
                 try await locationDataSource.start()
-                await withTaskGroup(of: Void.self) { group in
-                    group.addTask {
-                        for await location in locationDataSource.locations {
-                            lastLocationTimestamp = location.timestamp
-                            currentLocation = location
-                            await updateSceneView(for: location)
-                        }
-                    }
+                for await location in locationDataSource.locations {
+                    lastLocationTimestamp = location.timestamp
+                    currentLocation = location
+                    updateSceneView(for: location)
                 }
             } catch {}
         }
-        .toolbar {
-            ToolbarItem(placement: .bottomBar) {
-                if !calibrationViewIsHidden {
-                    calibrationView
+        .overlay(alignment: calibrationViewAlignment) {
```

Comment:
I think the calibration view should always be in the same spot, regardless of where the button is.

```suggestion
        .overlay(alignment: calibrationViewAlignment) {
            if configuration is ARWorldTrackingConfiguration,
               !calibrationViewIsHidden {
                if !viewModel.isCalibrating {
                    VStack {
                        Button {
                            withAnimation {
                                viewModel.isCalibrating = true
                            }
                        } label: {
                            Text("Calibrate calibrate")
                        }
                        .padding()
                        .background(.regularMaterial)
                        .clipShape(RoundedRectangle(cornerRadius: 10))
                        .disabled(!initialCameraIsSet)
                    }
                    .padding()
                }
            }
        }
        .overlay(alignment: .bottom) {
            if viewModel.isCalibrating {
                CalibrationView(viewModel: viewModel)
            }
        }
```

Reply:
Actually the VStack for the button is unnecessary:

```swift
                        Button {
                            withAnimation {
                                viewModel.isCalibrating = true
                            }
                        } label: {
                            Text("Calibrate calibrate")
                        }
                        .padding()
                        .background(.regularMaterial)
                        .clipShape(RoundedRectangle(cornerRadius: 10))
                        .disabled(!initialCameraIsSet)
                        .padding()
```

Reply:
Done.